dist/companies.json

```diff
@@ -9303,7 +9303,7 @@
         },
         "snap_technologies": {
             "name": "Snap Technologies",
-            "websiteUrl": "http://www.snap.com/",
+            "websiteUrl": "http://www.snapchat.com/",
```

Comment:
`companies.json` contains companies list merged with AdGuard companies from `adguard_companies.json`,  so the changes can only be added to `adguard_companies.json`, if I understand correctly

Reply:
I'll do it and we'll see what happens. :)